corehq/apps/app_manager/templates/app_manager/partials/releases/releases_table.html

```diff
@@ -69,7 +69,11 @@
       {% trans "Sorry, you don't have permission to do this action!" %}
     </span>
     <span data-bind="visible: buildErrorCode() !== 403">
-      {% trans "An error occurred. Reload the page and click Make New Version to try again." %}
+      <span data-bind="visible: genericErrorMessage, text: genericErrorMessage">
+      </span>
+      <span data-bind="visible: !genericErrorMessage()">
```

Comment:
I think you could simplify a little by deleting these two spans and instead adding a text binding to the outer span (the one that checks `buildErrorCode`) like `text: errorMessage` and then changing the line in the error handler to `self.errorMessage(xhr.responseText || gettext("An error occurred..."));`

Reply:
Just checking, is `responseText` definitely going to be blank or else something user-friendly, even if the request fails because of HQ maintenance or an expected error? We've had issues in other places where we end up printing HTML for the 500 page.

Reply:
Made two changes 
restructured the error message as suggested in https://github.com/dimagi/commcare-hq/pull/24916/commits/3069ce3b8def958012544258a5e4632795e014ae and then in https://github.com/dimagi/commcare-hq/pull/24916/commits/83aaa174867eed38e8a5727f1ececa8966a4b3ed ensured the message is set only when present so that the default message is shown.

But you are right, in case of exception it shows an HTML so going to update the approach here.

Reply:
Used a slightly different approach to not mess up with existing behaviour
https://github.com/dimagi/commcare-hq/pull/24916/commits/c06a741e158b7d87c26fb9f5825ce76ed9f0785e